include/maliput/geometry_base/road_geometry.h

```diff
@@ -168,6 +206,7 @@ class RoadGeometry : public api::RoadGeometry {
   std::vector<std::unique_ptr<Junction>> junctions_;
   std::vector<std::unique_ptr<BranchPoint>> branch_points_;
   api::BasicIdIndex id_index_;
+  std::unique_ptr<StrategyBase> strategy_;
```

Comment:
I am tempted to initialize `BruteForceStrategy` by default and let the user initialize other by demand. 
(Note that otherwise if InitailizeStrategy isn't called then the DoToRoadPosition and DoFindRoadPosition will throw because `strategy_` being nullptr).

It wouldn't impact current backends and it doesn't impose a method call when you simply want to use the standard one. 
Wdyt? @agalbachicar

Reply:
It's OK to keep the status quo but I think the default implementation must be the KDTree based one.

Reply:
The thing is that the KDTree strategy should be initialize after the RoadGeometry is populated. So, we need to user to trigger the initialization while with the brute force one we can just define it as default value of `strategy_`.

Reply:
Sounds good then.

Reply:
Done.